river/Output.zig

```diff
@@ -607,3 +607,104 @@ fn handleMode(listener: ?*c.wl_listener, data: ?*c_void) callconv(.C) void {
     self.arrangeViews();
     self.root.startTransaction();
 }
+
+/// Find the topmost surface under the output-local coordinates ox/oy
+/// returns the surface if found and sets the sx/sy parametes to the
+/// surface coordinates.
+pub fn surfaceAt(self: *Self, ox: f64, oy: f64, sx: *f64, sy: *f64) ?*c.wlr_surface {
+    // Check layers and views from top to bottom
+    const layer_idxs = @bitCast([4]c.zwlr_layer_shell_v1_layer, [_]c_int{
+        c.ZWLR_LAYER_SHELL_V1_LAYER_OVERLAY,
+        c.ZWLR_LAYER_SHELL_V1_LAYER_TOP,
+        c.ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM,
+        c.ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND,
+    });
```

Comment:
```suggestion
    const layer_idxs = [_]c.zwlr_layer_shell_v1_layer{
        .ZWLR_LAYER_SHELL_V1_LAYER_OVERLAY,
        .ZWLR_LAYER_SHELL_V1_LAYER_TOP,
        .ZWLR_LAYER_SHELL_V1_LAYER_BOTTOM,
        .ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND,
    });
```

Reply:
Oh thats great! Since the function got moved back to where it came from it is no longer necessary.